packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,922 @@
+---
+title: CAPTCHAs
+category: management
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" />
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+# CAPTCHAs API
+
+The Okta CAPTCHAs API provides operations to manage CAPTCHAs and org-wide CAPTCHA settings.
```

Comment:
Can we break these into a bulleted list that users can jump directly to? 

- CAPTCHAs
- Org-wide CAPTCHA settings